### DISCUSSION TITLE
BYG 1.20.. Let's talk about it!

### DISCUSSION THREAD

**Discussion by AOCAWOL** [CorgiTaco/BYG#1203] [Announcements]
Hello everyone! As we get ready for BYG 1.20.1 I wanted to make a blog post to talk about what you can expect from 1.20.1 BYG. Players might find themselves shocked at the changes and I wanted to break down our long-term plans and the reasoning behind what we've decided to do and lastly, what you can expect in future BYG updates moving forward!

### CONTENT CHANGES FOR 1.20.1

**GENERAL:**
-Addition of BYG Hanging Signs
-Reimplementation of the Tropical Village
-New biome exclusive Soundtracks for BYG Biomes (Can be controlled via Minecraft's Volume Settings)
Comments:
_"We're really excited to add the Hanging Signs as a new decoration block for you all to use in your world. The tropical village will also be re-added after its porting bugs from long ago. It will feature a fresh new look/design and some unique areas to it! We know a lot of people are asking if we are gonna add Chiseled Bookshelves for BYG Woodtypes, our answer right now is no. Chiseled Bookshelves are specialty blocks with a unique function and at this moment we don't feel any need to add more wood-type clutter than we already have."_

Removed the Following Blocks/Items:
_-Bulbis Phycelium
_-Pollen Block & Pollent Dust ✅
-Withering Oak ✅
-Torch Ginger ✅
-Blue & Yellow Netherrack Sets ✅
-Scoria Stone Set ✅
-Emeraldite Ore & Emeraldite Shards
-Ametrine Armor & Horse Armor ✅
-Pendorite Tools & Horse Armor ✅
-Chain Plating
-Nether Bristle ✅
-Subzero Ash ✅
-Subzero Ash Block ✅
-Hypogeal Imperium ✅
-Pervaded Netherrack ✅
-Glowstone Lantern ✅
-Glowstone Lamp ✅
-Blue Nether Ores ✅
-Shulkren Moss Blanket ✅
-Shulkren Moss Vine ✅
-Shulkren Phylium ✅
-Shulkren Fungus Set ✅
-Subzero Crystal Set ✅
-Frost Magma ✅
-Lament Woodtype ✅
-Lament Vegetation ✅
-Mycelium Netherrack ✅
-Overgrown Netherrack ✅
-Embur Nylium ✅
-Travertine Stone Set ✅
Comments:
_"This is a big list I know.. so ultimately from a development standpoint, we're not happy with the Nether Content and are doing a clean sweep. A lot of this content will see a return just in a more polished and flushed-out way. I know a lot of the content we are removing will be upsetting to our community but please have faith that you will see a lot of your favorite blocks come back soon! We also are really focused on removing clutter blocks & trying to clean up the random concepts in BYG and make them more solidified"_

SNIFFER:
-Lush Blocks, Peat, & Ether Phylium/Soil will be 'diggable' blocks for the Sniffer
-Lush Grass Blocks can enhance the speed at which Sniffer Eggs will Hatch (Like Moss Blocks)
-Protea Flower, Silver Vase Flower, & Black Rose will all be Sniffer Exclusive Digs now 
Comments:
_"We love the Sniffer so we wanted to give it some more uses for the time being but we also have some really awesome ideas for these creatures for future updates!"_

TRAIL RUINS:
-The following biomes will spawn Trail Ruins Structure: 
_Black Forest, Dacite Ridges, Forgotten Forest, Fragment Forest, Enchanted Tangle, Jacaranda Jungle, Tropical Rainforest, Weeping Witch Forest_
Comments:
_"These biomes are prime targets for Trail Ruins so hope you are able to find this structure a bit more find-able"_

**OVERWORLD:**
-Rename Cherry Blossom Forest -> ‘Sakura Grove’
-Rename Forgotten Forest -> ‘Enchanted Tangle’
-Rename Jacaranda Forest -> ‘Jacaranda Jungle’
-Rename Temperate Rainforest -> ‘Forgotten Forest’
-Renamed 'Joshua' Blocks & Items to 'Yucca'
Comments:
_"Ultimately we liked these names a lot better but they are still the same content you love!"_

CHERRY -> SAKURA:
-BYG Is Keeping our Cherry Woodtype but will be renaming it to Sakura Cherry. With the addition of the Vanilla Cherry Woodtype. The following is happening:
1. BYG Cherry is renamed to Sakura Cherry
2. Pink Cherry Leaves are being removed and replaced by Vanilla Cherry Leaves
3. We are keeping White Cherry Leaves, Saplings, & Foliage
4. We are adding Yellow Cherry Leaves, Saplings, & Foliage
5. Furthermore, the Sakura Grove (Previously Cherry Blossom Forest) is getting a tree overhaul/ground which will make the biome absolutely stunning!
Comments:
_"I am very happy to be keeping our BYG Cherry Wood in the mod. It's also super awesome that we can re-decorate the biome to be even prettier!"_

**NETHER**
-Removed 'Glowstone Gardens'
-Removed 'Crimson Gardens'
-Removed 'Subzero Hypogeal'
-Removed 'Weeping Mire'
-Removed 'Arisian Undergrowth'
Comments:
_"So, I know a lot of players are gonna be upset about this decision. It's a lot of content being removed in one bunch. Ultimately from a design standpoint, these biomes had a lot of issues with them and we are very unhappy with their current state. We are committing to our statement of Quality of Quantity. You will hopefully see some of these concepts/pallets come back in our Nether Update. There is a lot of new content being planned right now for the Nether that will fit a lot better theme-wise and concept-wise."_

**END**
-Removed Shulkren Forest ✅
Comments:
_"The End is also in an unpleasant state. However, we are a lot happier with the biomes/contents of this dimension. The Shulkren Forest is being removed due to a lack of quality within the biome itself. We hope to bring this back and lots of revamp to the End in our End Update!"_

---------------------------------------------------------------------------------------------------------------------------------------------------

### FUTURE CONTENT PLANS
Comments:
_"At the moment, we are taking things one step at a time. I don't want to share too much on our Nether & End plans at this time since it's still extremely experimental and on the back burner for now. I will share what we will be roughly doing for our first 1.20 content update. I can't give a time frame for when you will be able to see this content but hopefully within the 2023 year for sure!"_

**Update 1:**
GENERAL:

RUIN VILLAGES:
-Will contain suspicious Gravel and receive a complete overhaul

SNIFFER:
Add new Unique Sniffer Drops: 
  -Glowcane Shoot (4 Colors) 
  -Fluorescent Cattail Sprout
  -Imperium Sherd

NEW BLOCKS - GLOWCANE
-Glowcane Shoots can be dug up by Sniffers
-Shoot Colors: Green, Red, Yellow, Blue
-Shoots will Have Stages 0-2 then after Stage 2 will grow similar to Sugarcane
-Glowcane Shoots can be Grounded into Powders of their Respective Colors
-Crafting 1 Water Bottle with 4 Glowcane Powder will create a [Color] Glow Bottle
-Glow Bottles can be placed in the world as a Bioluminescent Light Source & Can be Hung as well similar to Lantern
-Powders can also make Dyes
-Powders can also change the color of Fluorescent Cattails
Comments:
_"ITS BACK! Glowcane will officially be returning to BYG after all these years. Glowcane can create some unique light sources and vegetation decor as well as unique interactions with the Fluorescent Cattails! We've made obtaining/farming them very unique but still easy while also giving you another reason to go find some Sniffers!"_

NEW BLOCK - FLUORESCENT CATTAIL
-Can be dug up by Sniffers
-Will function like Cattail Plants
-Cannot be used to craft Thatch
-Can be 'dyed' by Glowcane Powders
-Will emit a medium light source
-Will emit firefly-like particles 
-May attract nearby Aquatic Mobs to it for easy hunting/access
Comments:
_"A new unique plant block that can add some amazing biome/build ambiance for murky swamps & magical lakes. Fluorescent Cattails may receive additional interactions by the time of release so stay tuned!"_

NEW BLOCK - GLACIAL IMPERIUM:
-Collect 8 Imperium Sherds & 1 Nether Star to craft the Glacial Imperium
-Imperium Sherds can be rarely found in Archeology & Rarely in Sniffer Drops
-Now functions like a Beacon Alternative via the Material Pyramid System
-Bigger the Pyramid, the Larger the Radius (X, Y, Z) 
-No Resource/GUI is required for the block to function anymore
-As long as the Glacial Imperium can detect the Pyramid structure made of the allowed minerals (all just identical to the beacon) then it will stay active 
Comments:
_"The Hypogreal Imperium is back under a new name and easy-to-use function! We really love the concept of adding new uses for Nether Stars and encouraging our users to explore & become more involved in the Archeology aspect of Minecraft 1.20. Sniffers can very rarely dig these up so you have a better chance of obtaining them by excavating an area instead. This concept may also receive additional interactions/changes upon release!"_

NEW ARMOR/TOOL DIPS 
-Ametrine & Pendorite can now coat Netherrite Armor & Tools and apply a purely decorative coat over it
-Placing a Cauldron over a Lit Campfire and then right-clicking the cauldron with a Block of [Pendorite or Ametrine] will liquefy the material
-Players can do 3 dips per Block used
-Additional 'Armor Trims' can be put on Dipped Armor

ARMOR TRIMS
-The following is being considered for Armor Trims:
Ametrine, Pendorite, Therium

ARMOR TEMPLATES
-The following is being considered for Armor Templates:
Pumpkin Template (Pumpkin Patches)

**Comment by gittibene** [comment]
I appreciate that some biomes are going to be overhauled but i don't think removing so many blocks just to bring a lot of them back later is the right way. This is going to destroy our worlds and many builds for no reason.

If you go this route, please at least provide some kind of compatibility patch so our worlds still work with 1.20

**Reply by Alyysia** [reply]
I second this thought. Some of these blocks being removed are used extensively in my servers builds. Travertine especially is going to be pretty much impossible to replace with a different block set, without completely destroying the aesthetics of the builds. Glowstone lamps as well are pretty much irreplaceable, since redstone lamps need a power source.

**Reply by Erokaution** [reply]
I third this.
It generally just breaks updating from 1.19 to 1.20 for no real good reason.

**Reply by SylisMC** [reply]
If the mod is getting a full re-write. id much rather have a version available with less content sooner. as oppose to having to wait for an extra 6+ months to get the full mod available. You can always save your 1.19 worlds for later, or just start a new game. Think of it as an opportunity to add to your modpack

**Comment by MokahTGS** [comment]
Thanks for the update...good to know you aren't skipping 1.20.  Question:  Will BYG be supporting NeoForge?

**Reply by JT122406** [reply]
Yes

**Comment by Trickster-4** [comment]
Is it possible to give status updates on the 1.20 version?
so we know how is the development of it going. which stage it is, etc.
it would be fun to see the roadmap and each step being accomplished!

we're as excited for this update as you guys :)

please do not confuse this with ETA lol

**Reply by AwesomeDude091** [reply]
Just look at the checkmarks

**Comment by Fl414r** [comment]
So... 1.20.1 update is officially dead?

**Reply by ambientNexus** [reply]
Where'd you get that idea from?

**Reply by yavko** [reply]
Maybe because the branch hasn't had any commits in 2 months

**Reply by CorgiTaco** [reply]
we could be working on it in a secret repo ;)

**Reply by Fl414r** [reply]
> Where'd you get that idea from?

Cuz branch has no updates 2 months already (and this updates before is just some fixes from 1.19.2 branch) + if look at this list authors only deleted some stuff and added absolutely nothing.

**Reply by Fl414r** [reply]
> > Where'd you get that idea from?
> 
> Cuz branch has no updates 2 months already (and this updates before is just some fixes from 1.19.2 branch) + if look at this list authors only deleted some stuff and added absolutely nothing.

And of cource the reason that there already 7 months of ABSOLUTELY nothing!
No news, no progress, only updates on 1.19 and 1.18 versions.

**Reply by JT122406** [reply]
We have just been busy with life

**Comment by xuyin2333** [comment]
Will I be able to see the 1.20 update before the end of January? I'm really looking forward to this mod：）

**Reply by JT122406** [reply]
No

**Comment by Rebel459** [comment]
Quick question, I'm aware the update will take time, but will you still support at least an initial release for 1.20.1 or are you going to skip to later 1.20.x versions?

**Reply by JT122406** [reply]
Targeting 1.20.1

**Comment by PercyJames** [comment]
Will you be porting BYG to versions beyond 1.20?

**Reply by JT122406** [reply]
Yes

**Reply by PercyJames** [reply]
cool! I ask because I saw something somewhere that implied it was up in the air if you would support 1.21+

**Comment by kalethemonster** [comment]
hey, is there a general time frame for when byg 1.20.1 will release?

**Reply by Leafsong88** [reply]
As they Always says in Discord no ETA at the moment.

**Comment by allanonmage** [comment]
I am disappointed in the direction that you are taking this mod; you're basically killing it, which doesn't make any sense to me.

Also, specifically, this is a bad idea, and will cause confusion, after the anger, from users with borked worlds (due to an update from 1.19 to 1.20):

```
-Rename **Forgotten Forest** -> ‘Enchanted Tangle’
-Rename Jacaranda Forest -> ‘Jacaranda Jungle’
-Rename Temperate Rainforest -> **‘Forgotten Forest’**
```

**Reply by ambientNexus** [reply]
Why would you ever keep a modded world between major versions, instead of just grabbing schematics of your builds? Especially since you're just going to encounter issues either way due to how many other mods change things between major versions, or just don't get updates to later versions for extended if not indefinite periods of time.

**Reply by Fl414r** [reply]
> Why would you ever keep a modded world between major versions, instead of just grabbing schematics of your builds? Especially since you're just going to encounter issues either way due to how many other mods change things between major versions, or just don't get updates to later versions for extended if not indefinite periods of time.

Because some play on semi-vanilla, where there is really nothing of the global mods except BYG and for example Tom's Simple Storage or AE2, which never changes anything between versions.
And usually, just like in the regular vanilla version, they don't have to start over due to differences in mod versions...

**Comment by Trickster-4** [comment]
> I am disappointed in the direction that you are taking this mod; you're basically killing it, which doesn't make any sense to me.
> 
> Also, specifically, this is a bad idea, and will cause confusion, after the anger, from users with borked worlds (due to an update from 1.19 to 1.20):
> 
> ```
> -Rename **Forgotten Forest** -> ‘Enchanted Tangle’
> -Rename Jacaranda Forest -> ‘Jacaranda Jungle’
> -Rename Temperate Rainforest -> **‘Forgotten Forest’**
> ```

dude I understand where you're coming from but that's no way to talk to someone who is making something free for you.
blame minecraft and mojang doing the awful decision to make half baked updates every year. not corgi taco and the dev team.
just leave them alone, if you don't like how they're handling the mod just comission another modder to make the port exactly the way you want.

**Comment by mikaba-eu** [comment]
i have a question before i try the whole thing out. Is it difficult to edit the mod so that the blocks still work for 1.20.1? We created our building world with BYG and are now stuck on 1.19.2 because of this. We theoretically don't need the terrain generation of BYG, but our entire build world would be pretty broken without the mod. In itself, such a special fork shouldn't be difficult, right? Our build world is pregen and will not expend in the near future.

Besides the question if this is so easy, it is of course a question of license, we have a very small community ~15 players, if I would make a fork for it now, I would theoretically need permission from you because of the assets, right?

**Comment by Rebel459** [comment]
Yes, it would be difficult - I’d suggest you use a mod such as block swap to replace the removed blocks with different ones

**Comment by WigglyTBY** [comment]
At first, I was upset with the removal of some biomes, but I think it'll be worth it to see them revamped in the future. I'm excited to see what a lot of the affected biomes will become, because they all have a lot of potential and just need some fleshing out.

What I don't get is the removal of some block sets that aren't part of those biomes, especially ones from the Nether. I actually really like how the different materials make each biome different from the others, like blue netherrack and scoria stone. With the removal of ametrine and pendorite, it means there's less of a reason to visit the embur bog or viscal isles. Like, I get removing some biomes to revamp them, but why leave the other ones in if the main features of them are also gettting removed and revamped?

Either way, I know you're doing amazing work, and I look forward to seeing the amazing stuff coming at the other end of all the removals

**Comment by innocua** [comment]
Oh no, my beautiful Withering Oak! My favorite! Lol. Hey guys, I love all you do. I'm over here trying to set up my own new thing and see big changes are coming. That's awesome! I'm hyped. I've never gotten into the actual process of modding, but I'd like to help in any way I can. I did some programming way back in the 2000s so I'm not a total moron, but still way inexperienced. Not sure if I'm able, but I'm willing to try. Let me know! A no is understandable :)

**Comment by modpackmaker999** [comment]
HOW DO I PLAY THE ALPHA VERSION SOMEONE PLEASE TELL ME!!!!!!!
